omnibus/cookbooks/firezone/attributes/default.rb

```diff
@@ -381,17 +381,43 @@
 default['firezone']['robots_disallow'] = nil
 
 # ### Outbound Email Settings
-#
 # If none of these are set, the :sendmail delivery method will be used. Using
 # the sendmail delivery method requires that a working mail transfer agent
 # (usually set up with a relay host) be configured on this machine.
-#
-# SMTP will use the 'plain' authentication method.
-default['firezone']['from_email'] = nil
-default['firezone']['smtp_address'] = nil
-default['firezone']['smtp_password'] = nil
-default['firezone']['smtp_port'] = nil
-default['firezone']['smtp_user_name'] = nil
+default['firezone']['outbound_email']['from'] = nil
+default['firezone']['outbound_email']['provider'] = nil
+default['firezone']['outbound_email']['configs'] = {
+  smtp: {
+    # only relay is required, but you will need some combination of the rest
+    relay: 'smtp.example.com',
+    port: 25, # integer
```

Comment:
Just for security/posterity sake, do you think it might be better to use the SSL/TLS SMTP port as the example here? e.g. `465` or `587`. If a user *really* wants to use unencrypted SMTP they'd have to change it to `25` (as opposed to the other way around).

Reply:
Yep, I wasn't sure if we were going with bare minimum by default, as the comments previously said "plain by default". We can default these to secure options.

Reply:
Ah, yeah whoops, this config section was stubbed out when the Omnibus project was generated and was never actually hooked up to anything. Glad to see it's finally getting implemented.